### PR TITLE
git:Updated to 2.35.1

### PR DESCRIPTION
**Pull request by NoNameMan1231**

### CODE REVIEW

**Comment by NoNameMan1231** [comment]
Source is changed because the original download source is moved

**Review by thunder-coding — CHANGES_REQUESTED** [review]
Consider rebasing against master instead of merging the master branch into your branch. It doesn't cause problems on PRs but does on master branch

**Comment by NoNameMan1231** [comment]
> Consider rebasing against master instead of merging the master branch into your branch. It doesn't cause problems on PRs but does on master branch

Use `git rebase` in `master` ?

**Comment by thunder-coding** [comment]
> Use git rebase in master ?

```sh
git reset --hard <the commit you want to revert to>
git checkout master
git pull upstream master
git push
git checkout <this branch>
git rebase origin/master
git push -f
```

This should work here. If you want more information, you can check out the docs for the git subcommands used above

**Comment by NoNameMan1231** [comment]
> > Use git rebase in master ?
> 
> ```shell
> git reset --hard <the commit you want to revert to>
> git checkout master
> git pull upstream master
> git push
> git checkout <this branch>
> git rebase origin/master
> git push -f
> ```
> 
> This should work here. If you want more information, you can check out the docs for the git subcommands used above

But will it destroy this PR?

**Comment by thunder-coding** [comment]
> But will it destroy this PR?

There you go, you rebased it successfully! Congrats to you learning some new git subcommands!

**Review by thunder-coding — APPROVED** [review]

File: packages/git/build.sh [hunk 1]
```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://git-scm.com/
 TERMUX_PKG_DESCRIPTION="Fast, scalable, distributed revision control system"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=2.35.0
-TERMUX_PKG_SRCURL=https://www.kernel.org/pub/software/scm/git/git-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=47e677b475226857ceece569fb6ded8b85838ede97ae1e01bd365ac32a468fc8
+TERMUX_PKG_VERSION=2.35.1
+TERMUX_PKG_SRCURL=https://mirrors.edge.kernel.org/pub/software/scm/git/git-${TERMUX_PKG_VERSION}.tar.xz
```
Comment (thunder-coding) [inline packages/git/build.sh:6]:
```suggestion
TERMUX_PKG_SRCURL=https://mirrors.kernel.org/pub/software/scm/git/git-${TERMUX_PKG_VERSION}.tar.xz
``` 

Any reason for adding this `.edge` subdomain to the URL. Both seem to hold the same data. And the checksums are also unaltered

Comment (NoNameMan1231) [reply packages/git/build.sh]: Solved!

**Pull request merged**